jjb/vagrant-ceph.yaml

```diff
@@ -23,7 +23,7 @@
             name: BOX
             type: user-defined
             values:
-                - 'sub0/leap151'
+                - 'virt-appl/openSUSE-Leap-15.1'
```

Comment:
@colder-is-better could you please confirm this change is appropriate.

Reply:
@kshtsk Yes, that is the name of the box in https://github.com/openSUSE/vagrant-ceph/blob/master/config.yml (The "virt-appl" prefix implies that the Leap 15.1 box comes from the Virtualization Appliances repo.)